kratos/CMakeLists.txt

```diff
@@ -102,6 +102,7 @@ if(CMAKE_UNITY_BUILD MATCHES ON)
     set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/sources/model_part_io.cpp PROPERTIES SKIP_UNITY_BUILD_INCLUSION TRUE)
     set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/utilities/mortar_utilities.cpp PROPERTIES SKIP_UNITY_BUILD_INCLUSION TRUE)
     set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/utilities/exact_mortar_segmentation_utility.cpp PROPERTIES SKIP_UNITY_BUILD_INCLUSION TRUE)
+    set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/testing/distributed_test_case.cpp PROPERTIES SKIP_UNITY_BUILD_INCLUSION TRUE)
```

Comment:
can you try to revert this?
IMO this should not be related

Reply:
I tried a lot to revert it, but it is really strange...

Reply:
There are many cross-includes between node, geometry, .... Can we keep it as it is?, I tried again and is really troublesome...

Reply:
could you try with #8609? I suspect this might be the issue

Reply:
Yes I saw...

Reply:
I am unable to fix it... mysteries of the compiler